docs/configuration-guide.md

```diff
@@ -602,3 +602,123 @@ The section below describes all the available subtypes.
 |----------------|--------|-------------|---------------|
 | container-path | String | ""          |               |
 | secret-path    | String | ""          |               |  
+
+## Cert-Manager
+
+| Property                              | Type            | Description                                               | Default Value        |
+|---------------------------------------|-----------------|-----------------------------------------------------------|----------------------|
+| dekorate.certificate.name             | String          | Name of the certificate resource to be generated          | The application name |
+| dekorate.certificate.secretName       | String          | the name of the secret resource                           |                      |  
+| dekorate.certificate.issuerRef        | IssuerRef       | the reference to the issuer                               |                      |
+| dekorate.certificate.ca               | CA              | the CA issuer configuration                               |                      |
+| dekorate.certificate.vault            | Vault           | the Vault issuer configuration                            |                      |
+| dekorate.certificate.selfSigned       | SelfSigned      | the SelfSigned issuer configuration                       |                      |
+| dekorate.certificate.subject          | Subject         | the full X509 name specification                          |                      |
+| dekorate.certificate.commonName       | String          | common name to be used on the Certificate                 |                      |  
+| dekorate.certificate.duration         | String          | the lifetime of the Certificate                           |                      |  
+| dekorate.certificate.renewBefore      | String          | the renewal before duration timestamp                     |                      |  
+| dekorate.certificate.dnsNames         | String[]        | the list of [Subject Alternative Names](https://en.wikipedia.org/wiki/Subject_Alternative_Name) that should be included as part of the Order validation process. Find more information in [the Cert-Manager Dekorate](https://dekorate.io/docs/cert-manager#securing-resources) documentation.                           |                      |  
```

Comment:
`that should be included as part of the Order validation process` --> What is the order validation process ?

Reply:
This is a copy/paste from the Cert-Manager site, but I agree that without context, it's confusing. Removing.